io-engine/src/bin/io-engine-client/bdev_cli.rs

```diff
@@ -190,13 +195,14 @@ async fn destroy(mut ctx: Context, args: &ArgMatches<'_>) -> crate::Result<()> {
         .await
         .context(GrpcStatus)?;
 
-    let response = ctx
-        .bdev
-        .destroy(BdevUri {
-            uri: found.uri.clone(),
-        })
-        .await
-        .context(GrpcStatus)?;
+    // let response = ctx
```

Comment:
Shall we consider removing these commented lines ?

Reply:
I am yet to test all changes . Hence kept older calls. WIll test it tomorrow and remove.

Reply:
Done